pkg/kubelet/kubelet.go

```diff
@@ -3146,12 +3147,23 @@ func (kl *Kubelet) setNodeReadyCondition(node *api.Node) {
 	currentTime := unversioned.NewTime(kl.clock.Now())
 	var newNodeReadyCondition api.NodeCondition
 	if rs := kl.runtimeState.errors(); len(rs) == 0 {
-		newNodeReadyCondition = api.NodeCondition{
-			Type:              api.NodeReady,
-			Status:            api.ConditionTrue,
-			Reason:            "KubeletReady",
-			Message:           "kubelet is posting ready status",
-			LastHeartbeatTime: currentTime,
+		_, networkingCondition := api.GetNodeCondition(&node.Status, api.NodeNetworkingReady)
+		if (kl.cloud.ProviderName() == gce.ProviderName) && (networkingCondition == nil || networkingCondition.Status != api.ConditionTrue) {
```

Comment:
It's gated on GCE and I'm not sure how many people do it by hand vs. doing it by kube-push (however it's called right now). I don't have any idea on how to check if RC is in v1.3. I guess the best I can do is to have Kubelet write "NetworkingReadyCondition=false" on startup. It may lead to races though, which in turn will make us wait 5 minutes for the next RC pass...